apps/ui/src/components/schema/type.tsx

```diff
@@ -19,7 +19,7 @@ export function Type({ schemaId, type }: TypeProps) {
   }
 
   return (
-    <div className="flex flex-col text-sm">
+    <div className="flex flex-col text-sm  overflow-x-scroll custom-scrollbar p-2 bg-secondary-background rounded-2xl">
```

Comment:
Thanks for your work @ademajagon - can you please provide a short description why this has changed. And if you don't mind a screenshot on how it looks 

🚀 🚀 🙌